_opensearch/install/plugins.md

```diff
@@ -7,61 +7,173 @@ nav_order: 90
 
 # OpenSearch plugin installation
 
-You can install individual plugins on an OpenSearch cluster.
+You can install individual plugins for OpenSearch based on your needs. For information about available plugins, see [Available plugins]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#available-plugins).
 
----
+## Managing plugins
 
-#### Table of contents
-1. TOC
-{:toc}
-  
----
+OpenSearch uses a command line tool called `opensearch-plugin` for managing plugins. This tool allows you to:
+
+- [List]({{site.url}}{{site.baseurl}}/opensearch/install/plugins#list) installed plugins.
```

Comment:
I believe you need to add the complete link at the end of the url to get to the specific section. As in, "/plugins#list-installed-plugins"

Reply:
The link works fine for me - just tested it again locally to be extra sure.